### ISSUE TITLE
map not showing up

### ISSUE THREAD

**Issue by Lirang6** [AHAAAAAAA/PokemonGo-Map#2202]
I have done exerything according the Turtuiol and the map still not showing up not mater which browser i choose...

## Your Environment
* Version used: master version
* Environment name and version:  Python 2.7
* Operating System and version: desktop and mobil

**Comment by GreenflyM** [comment]
Having the same problem on mobile, but loading fine in firefox and chrome on desktop, did you try those?

**Comment by nepomuk13** [comment]
I think I have the same problem. Pokemon seems to load according to the console, interface is showing up (filtering options and so on) but not the map in itself

**Comment by haqthat** [comment]
Confirmed in chrome, and firefox, screenshot from chrome.

![image](https://cloud.githubusercontent.com/assets/1305090/17147801/54a4fef4-5332-11e6-8e47-58692b62cf33.png)

**Comment by Lirang6** [comment]
![image](https://cloud.githubusercontent.com/assets/20610887/17149021/18f6c618-5372-11e6-9f06-a240264b2cc9.png)
 
this is my problem... no map at all! what can i do?

**Comment by ricohumme** [comment]
Did you get your own Google Maps API key set up ?
[https://github.com/AHAAAAAAA/PokemonGo-Map/wiki/Google-Maps-API:-a-brief-guide-to-your-own-key](url)

**Comment by Gaudon** [comment]
I frequently have this issue as well, have to ctrl+f5 a lot to get it to work but then no pokemon actually show on the map heh.

**Comment by devinjparsons** [comment]
yeah looks like you need to make sure you have your gmaps api key in there

**Comment by GreenflyM** [comment]
I set up my own API key and even generated a second one with a separate account to test this. I get a blank screen with the options menu no matter what browser I use on ios. It works with chrome and firefox on desktop, but safari displays a blank screen with the options menu also.

**Comment by ricohumme** [comment]
I also have it working on Chrome, but IE is a no go. Only displays the black header and that is as far as it goes. Didn't check the console though to check if any errors are shown.

**Comment by Gaudon** [comment]
I have my own maps api key yeah, and chrome / firefox do not produce errors.. Just get a title menu bar at times and nothing else.. It's very unreliable.

**Comment by ricohumme** [comment]
In IE it shows the error InvalidValueError: initMap is not a function
Probably due to the Google maps API not being correctly loaded somehow

**Comment by haqthat** [comment]
Might have found the issue.

The error we were getting was: 
![image](https://cloud.githubusercontent.com/assets/1305090/17178466/9361519a-53e3-11e6-8a05-7d0ce30c3607.png)

After looking at the source, I saw: <script defer="" src="https://maps.googleapis.com/maps/api/js?key=credentials.json&amp;callback=initMap&amp;libraries=places,geometry"></script>

It appears, that using the "-k" flag is now expecting us to supply the actual KEY and not the credentials file to read the key out of.

I replaced "-k credentials.json" on the command line with "-k MY_API_KEY" and now everything is working.

**Comment by Gaudon** [comment]
I was already supply my key in both the config.ini file as well as the -k parameter and I still get no map loading more often than not. Have to force refresh on 15+ machines that I know are attempting to access the website (co-workers, friends).

**Comment by danipaan** [comment]
Same problem here as well. Works fine in Chrome (with own Google Api key, same as i used working in v1)

On iPhone map is not showing in either safari or chrome. On desktop only showing in Chrome, not in IE

What to do?

**Comment by JonahAragon** [comment]
I can replicate this issue in Safari on iOS. The top bar seems to work but the entire map area is blank.

I can confirm in my testing that everything works normally on Chrome for iOS, Chrome for Android, and Internet (Samsung's built in internet browser on the Galaxy S7). Additionally, it works on Chrome and Vivaldi on Windows 10.
